branding/celo/src/icons/Logo.tsx

```diff
@@ -3,46 +3,28 @@ import { View, ViewStyle } from 'react-native'
 import colors from 'src/styles/colors'
 import Svg, { Path } from 'svgs'
 
-export enum LogoTypes {
-  COLOR = 'COLOR',
-  DARK = 'DARK',
-  LIGHT = 'LIGHT',
-}
-
 interface Props {
-  height?: number
-  type?: LogoTypes
+  size?: number
+  color?: string
   style?: ViewStyle
   testID?: string
 }
 
-export default function Logo({ height = 25, type = LogoTypes.COLOR, testID, style }: Props) {
-  let mainColor
-  switch (type) {
-    case LogoTypes.DARK:
-      mainColor = colors.black
-      break
-    case LogoTypes.LIGHT:
-      mainColor = colors.white
-      break
-    default:
-      break
-  }
-
+export default function Logo({ size = 25, color, testID, style }: Props) {
   return (
     <View testID={testID} style={style}>
-      <Svg width={height} height={height} viewBox="0 0 25 25" fill="none">
+      <Svg width={size} height={size} viewBox="0 0 25 25" fill="none">
```

Comment:
This view box is different than the ValoraV. Is that ok?

Reply:
yes that should be fine